rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -472,3 +474,31 @@ def test_price_for_custom_assets(inquirer, database, globaldb):
     )
     inquirer.remove_cached_current_price_entry((asset, A_USD))
     assert inquirer.find_usd_price(asset) == Price(FVal(10))
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_coingecko_handles_rate_limit(inquirer):
+    """
+    Test that the mechanism to ignore coingecko when the user gets rate limited works as expected
+    """
```

Comment:
If it fits in one line please keep it 

```suggestion
    """Test that the mechanism to ignore coingecko when the user gets rate limited works as expected"""
```

Reply:
It doesn't fit in one line with the `"""`